carmajava/launch/cadillac_config/docker-compose.yml

```diff
@@ -0,0 +1,28 @@
+#  Copyright (C) 2018 LEIDOS.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License"); you may not
+#  use this file except in compliance with the License. You may obtain a copy of
+#  the License at
+# 
+#  http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+#  WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+#  License for the specific language governing permissions and limitations under
+#  the License.
+
+version: '2'
```

Comment:
What does this version refer to?  May be worthy of a comment to describe its meaning.

Reply:
It's the docker-compose YAML spec version, there's version 1 and version 3 as well and they all support slightly different sets of options and configurations. I can make a note